### ISSUE TITLE
Vertical lines (aka bars) within included content breaks table

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor#1421]
Vertical line (aka bar) characters within content included using the include directive inside a table cell breaks the table.

**Comment by mojavelinux** [comment]
We need to think about the best way to handle this problem without requiring a change to the included content (which might be coming from source code, for instance).

**Comment by mojavelinux** [comment]
The current workaround is to replace these characters in the source code with `{vbar}` and, if necessary, enable attribute substitutions on the block in which they are used. This is not a nice solution, but it's the only one atm.

**Comment by mojavelinux** [comment]
There are at least two solutions we could pursue:

* allow table cell separators to be escaped with a backslash
* only stop at a cell separator that starts at the beginning of the line (optionally proceded by colspec metadata) for AsciiDoc table cells

**Comment by mojavelinux** [comment]
Another workaround is to change the table cell (aka record) separator.

```
[cols="1,1a",separator=¦]
|===
¦Pipe output to vim
¦
----
asciidoctor -o - -s test.adoc | view -
----
|===
```

This workaround still requires that you be aware of content in the table, but at least you don't have to touch that content. Plus, you have the option of choosing a character (such as a broken bar) that is very unlikely to appear in the content.

The workaround of setting a user-defined separator definitely needs to be documented.

**Comment by stephenhay** [comment]
Is the user manual the only place this needs to be documented?

**Comment by mojavelinux** [comment]
Yes, the user manual is the canonical source of documentation about Asciidoctor features in general.
